frontend/src/console/components/PreviewComponent/styles.scss

```diff
@@ -0,0 +1,110 @@
+.theme-preview {
+  border: 2px solid black;
```

Comment:
Can you change this to `1px`? The tick border doesn't look good in HD and mobile screens

Reply:
Done!